edb/server/mng_port/port.py

```diff
@@ -35,6 +35,7 @@
 
 
 logger = logging.getLogger('edb.server')
+telemetry = logging.getLogger('edb.server.telemetry')
```

Comment:
ditto, let's avoid using `telemetry` for anything *but* actual telemetry that sends data back to EdgeDB. We don't have that yet (and it's unclear if we ever will).